ci/scripts/verible-format.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# A wrapper around Verible format, used for CI.
+
+set -e
+
+util/verible-format.sh
+
+if [ $? != 0 ]; then
+  echo "Verilog format with Verible failed. Run 'util/verible-format.sh' to check and fix all errors."
+  echo "This flow is currently experimental and failures can be ignored."
```

Comment:
Prefix these messages with `##vso[task.logissue type=error]` to make them show up in the checks UI.

Reply:
Done.